toolbox/fdc3-conformance/Open-Tests.md

```diff
@@ -0,0 +1,66 @@
+- `AFailsToOpenB2`:  **A** uses `fdc3.open({name: ‘<non existent app>’})`  ![1.2](https://img.shields.io/badge/FDC3-1.2-green)
+- `AFailsToOpenB3`:  **A** uses `fdc3.open({name: ‘<non existent app>’, appId: “<non existent app id>”})`  ![1.2](https://img.shields.io/badge/FDC3-1.2-green)
+
+## A Opens B With Context
+
+| App | Step            | Description                                              |
+|-----|-----------------|----------------------------------------------------------|
+| A   | 1. Opening App     |App A opens app B with an `fdc3.instrument` context data item |
+| B   | 2. Context present | Add an untyped context listener via: <br/>`fdc3.addContextListener(null, handler)` <br /> B receives a matching piece of context data from A |
+
+- `AOpensBWithContext1`:  **A** uses `fdc3.open(‘app B Name', ctx)` ![1.2](https://img.shields.io/badge/FDC3-1.2-green) ![2.0](https://img.shields.io/badge/FDC3-2.0-blue)
+- `AOpensBWithContext2`:  **A** uses `fdc3.open({name: “<app B Name>”}, ctx)` ![1.2](https://img.shields.io/badge/FDC3-1.2-green)
+- `AOpensBWithContext3`:  **A** uses `fdc3.open({name: “<app B Name>”, appId: “<app B ID>”}, ctx)` ![1.2](https://img.shields.io/badge/FDC3-1.2-green)
+- `AOpensBWithSpecificContext`: Perform above but replace **B**s call with `fdc3.addContextListener('fdc3.instrument', handler)` ![1.2](https://img.shields.io/badge/FDC3-1.2-green) ![2.0](https://img.shields.io/badge/FDC3-2.0-blue)
+- `AOpensBMultipleListen`:  Perform `AOpensBWithSpecificContext` but **B** should perform an additional `fdc3.addContextListener('fdc3.contact', handler)` prior to the existing `addContextListener` for `fdc3.instrument`.  The correct context listener should receive the context, and the promise completes successfully.  ![1.2](https://img.shields.io/badge/FDC3-1.2-green) ![2.0](https://img.shields.io/badge/FDC3-2.0-blue)
+
+
+## Wrong Context ![2.0](https://img.shields.io/badge/FDC3-2.0-blue)
+
+| App | Step            | Description                                                                                                                   |
+|-----|-----------------|-------------------------------------------------------------------------------------------------------------------------------|
+| A   | 1. Opening App     | A opens B using `fdc3.open(‘app B Name', ctx)`, but passes a context type that app B is NOT going to add a listener for (including NOT adding a listener for type `null`)                                                                |
+| B   | 2. Context not received | App B adds a context listener for a different type than the one A is sending.  <br/>Confirm that NO context is received.                         |
+| A   | 3. Promise         | Receives a rejection from the open promise with `AppTimeout` from <br>https://fdc3.finos.org/docs/api/ref/Errors#openerror |
+
+- `AOpensBWithWrongContext`: Perform above test
```

Comment:
This test can actually be applied to both versions, the error has existed for a long time (although docs are better in 2.0). I think we just missed it in the first 1.2 version. 

However, it also varies in the calling syntax for the app - somehow you've ended up with the 1.2 syntax only. 

Removing from here and placing it above where its easier to referencethe existing definitions

```suggestion
```